### PR TITLE
Week 10: Nadia Lefebvre

### PR DESCRIPTION
**Pull request by nadialefebvre**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/RandomChoices.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+let friendName = ""
+let sport = ""
+let ingredientOne = ""
+let ingredientTwo = ""
+let phone = ""
+
+export const RandomGender = () => {
+  const genderOptions = ["feminine", "masculine", "non-binary"]
+  if (gender === "") {
+    gender = genderOptions[Math.floor(Math.random() * (genderOptions.length))]
```
Comment (TereseClaesson) [inline code/src/components/RandomChoices.jsx:17]: Smart to use a component for random answers and to use math.random for this :)

File: code/src/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+.back-button-box {
+  margin-top: 40px;
+}
+
+.back-button-box button {
+  color: #fff;
+  background-color: grey;
+}
+
+.error-message {
```
Comment (TereseClaesson) [inline code/src/styles/index.css:64]: Logical choice of color

File: code/src/styles/index.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  padding-bottom: 20px;
+}
+
+.checkboxes {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 20px;
+}
+
+.single input[type="checkbox"] {
```
Comment (TereseClaesson) [inline code/src/styles/index.css:109]: Nicely done with the checkbox

File: code/src/styles/typewriter.css [hunk 4]
```diff
@@ -0,0 +1,213 @@
+@keyframes movePaper {
+  0% { transform: translate(-50%, -50%); }
+  100% { transform: translate(calc(-50% - 6ch), -50%); }
+}
+
+@keyframes moveTypewriter {
```
Comment (TereseClaesson) [inline code/src/styles/typewriter.css:6]: I like the typwriter animation, a nice touch!

File: code/src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
```
Comment (TereseClaesson) [inline code/src/App.jsx:1]: Clean entry point

**Review by TereseClaesson — APPROVED** [review]
A clean and well structured code, with good consistency throughout the project! 

Great work using aria-labels, tabindex and for acessibility purpose. Good naming using PascalCase.  

I liked that you used one of your own stories in the end and merged it with the answers from the survey. Really nice touch!

There where a tiny little spelling mistake I noticed when reading the story : 
“As usual, Sheis ,what about you?” answers the door attendant.  
The highligthed answerword after As usual in this case She is need a space between them ;) 

I'm really impressed by this react project Nadia! Excellent accomplish!

File: README.md [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
+
+I had some difficulties due to the ambitious concept that I chose to create, but at the end I was able to solve most of them or find some workaround until I learn more about React. If I had more time, I would have customised the radio buttons/checkboxes but it seemed a very minor styling detail that I decided to let go since it was still looking good with the minimalist design that I made.
+
+All in all, I am very proud of what I was able to do with this second project using React and the first one with State hook.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Project deployed here: [Story Factory](https://story-factory.netlify.app/)
```
[the remark this answers is not in the document]
Comment (nadialefebvre) [reply README.md:15]: Yes, it was intended :)

File: code/public/index.html [hunk 7]
```diff
@@ -1,34 +1,28 @@
...
+  <meta property="og:title" content="Story Factory | get a customised short story">
+  <meta property="og:description" content="Story Factory | get a customised short story based on your choices or random choices">
+  <meta property="og:image" content="https://story-factory.netlify.app/story-factory.jpg">
+  <meta name="twitter:card" content="summary_large_image">
+  <meta property="twitter:image" content="https://story-factory.netlify.app/story-factory.jpg">
+  <meta property="twitter:domain" content="story-factory.netlify.app">
+  <meta property="twitter:url" content="https://story-factory.netlify.app/">
+  <meta name="twitter:title" content="Story Factory | get a customised short story">
+  <meta name="twitter:description" content="Story Factory | get a customised short story based on your choices or random choices">
+  <meta name="twitter:image" content="https://story-factory.netlify.app/story-factory.jpg">
```
[the remark this answers is not in the document]
Comment (nadialefebvre) [reply code/public/index.html:19]: These tags are for sharing on social media, so you have a nice little card with an image, etc. See here: https://www.opengraph.xyz/

**Pull request closed without merging**